src/Workspaces/Core/Portable/Workspace/Host/SolutionServices.cs

```diff
@@ -53,5 +53,8 @@ public LanguageServices GetLanguageServices(string languageName)
 
         public TLanguageService GetRequiredLanguageService<TLanguageService>(string language) where TLanguageService : ILanguageService
             => this.GetLanguageServices(language).GetRequiredService<TLanguageService>();
+
+        internal IEnumerable<T> FindLanguageServices<T>(HostWorkspaceServices.MetadataFilter filter)
+            => _services.FindLanguageServices<T>(filter);
```

Comment:
This is a very specialized helper that HostWorkspaceServices has; it's public and used only in this MSBuildWorkspace place. I'm just keeping it internal.